generate-component-docs/__snapshots__/generate.test.ts.snap

```diff
@@ -0,0 +1,6618 @@
+      "type": Object {
+        "alias": "ResponsiveProp",
+        "params": Array [
+          Object {
+            "type": "union",
+            "types": Array [
+              "\\"none\\"",
+              "\\"xxsmall\\"",
+              "\\"xsmall\\"",
+              "\\"small\\"",
+              "\\"medium\\"",
+              "\\"large\\"",
+              "\\"xlarge\\"",
+              "\\"xxlarge\\"",
+              "\\"gutter\\"",
+            ],
+          },
+        ],
+        "type": "alias",
+      },
+    },
+    "paddingTop": Object {
+      "propName": "paddingTop",
+      "required": false,
+      "type": Object {
+        "alias": "ResponsiveProp",
+        "params": Array [
+          Object {
+            "type": "union",
+            "types": Array [
+              "\\"none\\"",
+              "\\"xxsmall\\"",
+              "\\"xsmall\\"",
+              "\\"small\\"",
+              "\\"medium\\"",
+              "\\"large\\"",
+              "\\"xlarge\\"",
+              "\\"xxlarge\\"",
+            ],
+          },
+        ],
+        "type": "alias",
+      },
+    },
+    "pattern": Object {
+      "propName": "pattern",
+      "required": false,
+      "type": "string",
+    },
+    "placeholder": Object {
+      "propName": "placeholder",
+      "required": false,
+      "type": "string",
+    },
+    "playsInline": Object {
+      "propName": "playsInline",
+      "required": false,
+      "type": "boolean",
+    },
+    "poster": Object {
+      "propName": "poster",
+      "required": false,
+      "type": "string",
+    },
+    "prefix": Object {
+      "propName": "prefix",
+      "required": false,
+      "type": "string",
+    },
+    "preload": Object {
+      "propName": "preload",
+      "required": false,
+      "type": "string",
+    },
+    "property": Object {
+      "propName": "property",
+      "required": false,
+      "type": "string",
+    },
+    "radioGroup": Object {
+      "propName": "radioGroup",
+      "required": false,
+      "type": "string",
+    },
+    "readOnly": Object {
+      "propName": "readOnly",
+      "required": false,
+      "type": "boolean",
+    },
+    "rel": Object {
+      "propName": "rel",
+      "required": false,
+      "type": "string",
+    },
+    "required": Object {
+      "propName": "required",
+      "required": false,
+      "type": "boolean",
+    },
+    "resource": Object {
+      "propName": "resource",
+      "required": false,
+      "type": "string",
+    },
+    "results": Object {
+      "propName": "results",
+      "required": false,
+      "type": "number",
+    },
+    "reversed": Object {
+      "propName": "reversed",
+      "required": false,
+      "type": "boolean",
+    },
+    "role": Object {
+      "propName": "role",
+      "required": false,
+      "type": "string",
+    },
+    "rowSpan": Object {
+      "propName": "rowSpan",
+      "required": false,
+      "type": "number",
+    },
+    "rows": Object {
+      "propName
```

Comment:
`ReactNode` might be something else to not unpack, from a docs site perspective things can get noisy here.